developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
+
+#### Profile picture (PFP) NFTs
+
+A PFP NFT has a unique token data ID and picture for each token. There are generally no printing editions of this NFT. Most NFT collections on NFT marketplaces are of this kind. They are generally proofs of ownership of an art piece.
+
+In this tutorial, we describe how to create and mint event ticket NFTs.
+
+### Accounts
+
+When you are minting an NFT, the NFT is stored under your [account](../../concepts/accounts.md) address. When you submit a transaction, you sign the transaction. Find your account configuration information in `.aptos/config.yaml` relative to where you run `aptos init` (below).
+
+[Resource accounts](../resource-accounts.md) allow the delegation of signing transactions. You create a resource account to grant a signer capability that can be stored in a new resource on the same account and can sign transactions autonomously. The signer capability is protected as no one has access to the private key for the resource account.
+
+### Initialization
+
+The `init_module` of [`minting.move`](https://github.com/aptos-labs/nft-tutorial/blob/main/sources/minting.move) always gets called and run when the module is published:
+
+```shell
+    fun init_module(resource_account: &signer) {
+        let resource_signer_cap = resource_account::retrieve_resource_account_cap(resource_account, @source_addr);
+        let resource_signer = account::create_signer_with_capability(&resource_signer_cap);
+```
+
+The `mint_nft` function of [`minting.move`](https://github.com/aptos-labs/nft-tutorial/blob/main/sources/minting.move) gets a collection and creates a token.
+
+With the resulting TokenData ID, the function uses the resource signer of the module to mint the token to the `nft-receiver`.
+
+For example:
+```shell
+    public entry fun mint_nft(receiver: &signer) acquires ModuleData {
+        let receiver_addr = signer::address_of(receiver);
+```
+
+### Signing
+
+The only argument taken by `mint_nft` is `receiver`. Any `entry fun` will take as the first parameter the type `&signer`. In both Move and Aptos, whenever you submit a transaction, the private key you sign the transaction with automatically makes the associated account the first parameter of the signer.
+
+You can go from the signer to an address but normally not the reverse. So when claiming an NFT, both the private keys of the minter and receiver are needed, as shown in the instructions below.
+
+In the `init_module`, the signer is always the account uploading the contract. This gets combined with:
+
+```shell
+        token::create_collection(&resource_signer, collection, description, collection_uri, maximum_supply, mutate_setting);
+
+```
+Where the `&resource_signer` is the first parameter, defined previously as a new address that has all of the attributes of the original account plus signer capability. See:
```

Comment:
I am not sure if I am following yhid line 
`Where the `&resource_signer` is the first parameter, defined previously as a new address that has all of the attributes of the original account plus signer capability.`

Could you clarify?

Reply:
This comes from the discussion between Igor, Alden and Max. Perhaps this is better:

Where the `&resource_signer` is the first parameter and represents a resource account with a new address that has all of the attributes of the original account plus signer capability.

Or we can strike as confusing. WDYT?

Reply:
I removed this language.